Makefile

```diff
@@ -39,10 +21,13 @@ example-executor:
 	go build -o ${EXAMPLES}/$@ ${EXAMPLES}/example_executor.go
 
 format:
-	go fmt ${LIBS:%=$(PKG_PREFIX)/%}
+	gofmt -s -w .
+
+formatted:
+	! gofmt -s -d . 2>&1 | read
 
 vet:
-	go vet ${LIBS:%=$(PKG_PREFIX)/%}
+	go vet ./...
 
-test test.v:
-	flags=""; test "$@" != "test.v" || flags="-test.v"; pkg="${TEST}"; test -n "$$pkg" || pkg="${LIBS:%=$(PKG_PREFIX)/%}"; go test $$pkg $$flags
+test: formatted vet
```

Comment:
Fair point regarding the expectation for `make test`. Even though `make check` has also historical uses, I agree that it's more appropriate in this case.

There's no need to type `go test -v github.com/mesos/mesos-go/detector/zoo` since relative paths are accepted by the go tool, this would be enough: `go test -v ./detector/zoo`.